{{cookiecutter.project_name}}/Keras_Tensorflow/03_BuildImage.ipynb

```diff
@@ -438,7 +438,7 @@
    "formats": "ipynb"
   },
   "kernelspec": {
-   "display_name": "Python 3",
```

Comment:
Any reason we are changing versions here? Has this been fully vetted as to not cause any more issues?

Reply:
Jupyter notebook picked this up from the virtual environment created for this HP.

Reply:
IOW, it is what it should have been?

Reply:
I went in and put it back to match the other notebooks